frontend/src/containers/LearnMore.js

```diff
@@ -0,0 +1,147 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import LoginTopBar from '../containers/LoginTopBar';
+import PublicFooter from '../components/PublicFooter';
+import Grid, { Column } from '../components/Grid';
+
+export class LearnMore extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className='LearnMore'>
+          <LoginTopBar />
+          <div className='LearnMore-container'>
+            <section className='LearnMore-Section1'>
+              <div className='LearnMoreHeader'>Discover a new form of association</div>
+              <div className='LearnMoreHeader -faded'>Transparent by design.</div>
+              <div className='-img'></div>
+            </section>
+            <section className='LearnMore-Section2'>
+              <div className='LearnMoreHeader'>Open Your Finances To Your Community</div>
```

Comment:
Latter. No quotes anywhere on the page. That was more to convey the exact phrase.